.circleci/config.yml

```diff
@@ -8,6 +10,9 @@ executors:
     environment:
         JVM_MEMORY: 3500M # Default JVM maximum heap limit
         LANG: en_US.UTF-8 # required by sbt when it sees boost directories
+  aws:
+    docker:
+    - image: cimg/python:2.7-node
```

Comment:
Is this just a basic image with Python 2.7?

Reply:
Yes.

Reply:
It's the newer 'cimg' instead of 'circleci' namespace python image with python 2.7 and node.js installed.  

See:
- https://circleci.com/docs/2.0/circleci-images/#next-gen-circleci-images  regarding 'cimg' vs 'circleci' dockerhub namespace
- https://hub.docker.com/r/cimg/python for a description of what the '2.7-node' tag means.